mapiproxy/dcesrv_mapiproxy.c

```diff
@@ -286,8 +287,6 @@ static void mapiproxy_op_unbind(struct dcesrv_connection_context *context, const
 		talloc_free(private);
 	}
 
-	talloc_free(context);
-
```

Comment:
That is fairly understandable at it also require some context from samba4 rpc server. In a nutshell, samba rpc_server is responsible for freeing the dcesrv_connection_context, not the endpoint. 

With the patch for samba4 that addresses the termination of connection in asynchronous emsmdb, it exposes this problem more radically and causes the process to crash with an attempt to double free talloc_free data.